opennms-container/minion/container-fs/entrypoint.sh

```diff
@@ -183,6 +184,10 @@ configure() {
   initConfig
   applyConfd
   applyOverlayConfig
+  chmod a+x "$MINION_PROCESS_ENV_CFG"
+  set -a
+  source "$MINION_PROCESS_ENV_CFG"
```

Comment:
What's the expected behavior if JAVA_OPTS is already set in the environment?

Reply:
My preference is to overwrite in the case we have been explicitly provided one via confd. This should only ever be set by a user intending to provide their own options in full. The plan is to move the config that dominion already sets in the environment to go through confd instead to preserve the existing defaults.